distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -1817,3 +1819,50 @@ Example:
     ]
 }
 ----
+
+==== Attribute Injection Definition
+
+Attribute injections are defined attributes that will be injected into all metacard types or into specific metacard types. Essentially, they allow you to extend existing metacard types with new attributes.
+
+To define attribute injections, your definition file must have an `inject` key in the root object.
+
+[source,json]
+----
+{
+    "inject": [...]
+}
+----
+
+The value of `inject` is simply a list of objects where each object contains the key `attribute` and optionally `metacardTypes`.
+
+[source,json]
+----
+{
+    "inject": [
+        {
+            "attribute": ...,
+            "metacardTypes": [...]
+        }
+    ]
+}
+----
+
+The value corresponding to the `attribute` key is the name of the attribute to inject.
+
+The `metacardTypes` key is optional. If it is left out, then the attribute will be injected into every metacard type. It can be thought of as a 'global' attribute injection. If the `metacardTypes` key is included, then its value must be a list of strings where each string is the name of a metacard type. In this case, the attribute will be injected only into metacard types that match one of the types given in the list.
```

Comment:
Should there be a note here that injected attributes must have a corresponding definition in the attribute registry?